<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="UTF-8">
    <title>IO17 Immuno-oncology</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="css/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="css/cayman.css" media="screen">
  </head>
  <body>
    <section class="page-header">
      <h1 class="project-name">IO17 </br> Large-scale bioinformatics </br> for Immuno-Oncology</h1>
      <h2 class="project-tagline">Instituto Gulbenkian de CiÃªncia, Oeiras, Portugal<br>19-22 September 2017<br></h2>
      
    </section>

    <section class="main-content">
      <h1>
<a id="overview" class="anchor" href="#overview" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Course Description</h1>

<p>The immuno-oncology approach leverages on the unique capability of the immune system to recognize and kill tumour cells. This action is hampered by escape mechanisms put in place by tumour cells like, for instance, the engagement immune checkpoints, i.e. inhibitory molecules that modulate the amplitude and duration of immune responses. Immunotherapies that block checkpoint molecules are amongst the most promising approaches in immuno-oncology for the enhancement of antitumour immunity. Thanks to high-throughput technologies, such as next-generation sequencing (NGS) and proteomics, we have now access to large-scale tumour data that can be used to investigate the interplay between tumour and immune cells and the role of the immune system in tumour progression and response to therapy. In this course, you will learn to use bioinformatics tools and mathematical modelling techniques operating on high-throughput tumour data, in order to extract features that can be used to characterise this complex tumour-immune cell interface, such as:</p>
<ul>
<li>Tumour antigens recognized by T cells</li>
<li>Tumour-infiltrating immune cells</li>
<li>Deregulated signalling pathways in cancer and immune cells</li>
</ul>
A fully practical, hands-on approach will ensure that the newly acquire skills can be used with a great deal of autonomy.
</p>

<h1>
<a id="target-audience" class="anchor" href="#target-audience" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Target audience</h1>

<p>Motivated researchers, clinicians, and students who want to gain an understanding on how bioinformatics tools and simple (logic-based) modelling approaches can be used to investigate the tumour-immune cell interface and its underlying signalling pathways from high-throughput data.
</p>

<h1>
<a id="pre-requisites" class="anchor" href="#pre-requisites" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Pre-requisites</h1>

<p>Programming/scripting skills are helpful, but not mandatory. An understanding of elementary operations with R will be required. Elementary command line instructions in UNIX will be used, so minimal familiarity with navigation in directory trees, copying files and folders, etc. will be needed.
</p>

<h1>
<a id="program" class="anchor" href="#program" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Program</h1>

<p>A detailed program is reported <a href="https://github.com/FFinotello/Immuno-Oncology/blob/master/Program.md">here</a>.
</p>

<h2>
<a id="authors-and-contributors" class="anchor" href="#authors-and-contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Authors and Contributors</h2>
<p>Francesca Finotello, Federica Eduati, and Pedro L. Fernandes</p>
      <footer class="site-footer">

        <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
      </footer>

    </section>

  
  </body>
</html>
